services/blockchain-app-registry/shared/utils/fsUtils.js

```diff
@@ -179,6 +179,40 @@ const rename = async (oldName, newName) => new Promise((resolve, reject) => {
 	});
 });
 
+const deleteFolderIfEmpty = async (folderPath) => {
+	const files = await fs.promises.readdir(folderPath);
```

Comment:
We already have method implemented to get all files. We can re-use that method here:

```suggestion
	const files = await getFiles(folderPath);
```